cosmic_ray/cli.py

```diff
@@ -56,7 +56,7 @@ def handle_help(config):
     if not command:
         options = OPTIONS
     elif command not in COMMAND_HANDLER_MAP:
-        LOG.error('"{}" is not a valid cosmic-ray command'.format(command))
+        LOG.error('"%s" is not a valid cosmic-ray command' % command)
```

Comment:
I thought the best practice here was `LOG.error('"%s' is not a valid cosmic-ray command', command)`, i.e. to pass interpolation values to `error()` rather than filling in template values ourselves. This way the interpolation never happens unless the logger actually needs to generate the display value.